### PR TITLE
LPS-152581 [Raylife AP] create reusable charts components

### PR DESCRIPTION
**Pull request by silasbsi**
Story: [LPS-152581](https://issues.liferay.com/browse/LPS-152581)

### CODE REVIEW

**Comment by silasbsi** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/applications-status/index.js [hunk 1]
```diff
@@ -115,74 +111,20 @@ export default function () {
...
-
-							<div
-								className="font-weight-bolder"
-								style={{fontSize: '14px'}}
-							>
-								{column[1]}
-							</div>
-						</div>
-					))}
+			{chartData.columns.length === 0 && loadData && (
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/applications-status/index.js:123]:
```suggestion
			{chartData?.columns?.length === 0 && loadData && (
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/policies-total-active/index.js [hunk 2]
```diff
@@ -125,70 +116,15 @@ export default function () {
...
-									{column[2]}
-								</span>
-							</div>
-
-							<span className="font-weight-bolder">
-								{column[1]}
-							</span>
-						</div>
-					))}
+			{chartData.columns.length > 0 && (
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/policies-total-active/index.js:119]:
```suggestion
			{chartData?.columns?.length > 0 && (
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/policies-total-active/index.js [hunk 3]
```diff
@@ -125,70 +116,15 @@ export default function () {
...
+				<DonutChart chartData={chartData} title={chartTitle} />
+			)}
+
+			{chartData.columns.length === 0 && loadData && (
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/policies-total-active/index.js:123]: Same here 
```suggestion
			{chartData?.columns?.length === 0 && loadData && (
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/applications-products/index.js [hunk 4]
```diff
@@ -105,70 +96,15 @@ export default function () {
...
-							<span className="font-weight-bolder">
-								{column[1]}
-							</span>
-						</div>
-					))}
+			{chartData.columns.length > 0 && (
+				<DonutChart chartData={chartData} title={chartTitle} />
+			)}
+
+			{chartData.columns.length === 0 && loadData && (
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/applications-products/index.js:103]: Same here

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job!

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Review by joanagrego — APPROVED** [review]

**Review by eliasantosf — APPROVED** [review]
LGTM! Thanks @silasbsi

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**